corehq/apps/zapier/api/v0_5.py

```diff
@@ -107,7 +107,7 @@ def obj_get_list(self, bundle, **kwargs):
             ))
 
         for form_property in MAIN_FORM_TABLE_PROPERTIES:
-            if form_property.is_advanced:
+            if form_property.is_advanced or form_property.item.transform:
```

Comment:
this fixes one of the issues highlighted [here](https://cdn.zapier.com/storage/photos/f109159ec8f2cc0cdde9f8ae86a762ab.png) (by removing the field that wasn't getting transformed anyways)